build-system/global-configs/prod-config.json

```diff
@@ -25,7 +25,6 @@
   "amp-access-iframe": 1,
   "amp-ad-ff-adx-ady": 0.01,
   "amp-apester-media": 1,
-  "amp-auto-ads": 1,
```

Comment:
I know this won't be an issue anymore. But I assume presubmit check won't allow you to make the changes at the same time. Let me know if that's still the case. Thanks.

Reply:
i removed the check :-)

Reply:
Cool 👍